templates/taxi/confirm_delete.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  {% if object is manufacturer %}

```

Comment:
Using `is` for object type checking in Django templates might not work as expected. Consider implementing a custom template filter for more reliable type checking.